pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dao/JdbiTemplate.java

```diff
@@ -67,14 +67,12 @@ boolean update(@Bind("templateId") long templateId,
     int updateRevisionIdById(@Bind("templateId") long templateId,
                                 @Bind("revisionId") long revisionId);
 
-    @SqlQuery("SELECT tmpl.template_id AS id, "
-                + " tmpl.template_code AS code, "
-                + " type.template_type_code AS type, "
-                + " tmpl.template_text AS text, "
-                + " tmpl.revision_id "
-                + "FROM template AS tmpl "
-                + "JOIN template_type AS type ON type.template_type_id = tmpl.template_type_id "
-                + "WHERE tmpl.template_id = :templateId ")
+    @SqlQuery("SELECT tmpl.*, "
```

Comment:
tmpl.* already selects template_id  but Mapper probably ignores it because column_name doesn't match.
Explicit selection of column names  is more cleaner but more lines though.